core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -471,6 +471,15 @@ package object config {
         "cache block replication should be positive.")
       .createWithDefaultString("30s")
 
+  private[spark] val STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH =
+    ConfigBuilder("spark.storage.decommission.fallbackStorage.path")
+      .doc("The location for fallback storage during block manager decommissioning." +
```

Comment:
nit: it's better to remind the user to enable TTL in the doc since Spark doesn't clean up the fallback storage.

Reply:
Sure, @zsxwing !